### DISCUSSION TITLE
windows 10 configuration file

### DISCUSSION THREAD

**Discussion by yylays** [BurntSushi/ripgrep#1650] [General] [answered]
wherever i set RIPGREP_CONFIG_PATH,  the powershell throw os error5 and refuse access to the .ripgreprc file, how can i resolve this problem? Thanks in advance.

**Comment by BurntSushi** [comment]
I'd love to help you, but there really aren't enough details here to go on. You say, for example that you get an error but don't provide the details of the error message. You also say that you "set RIPGREP_CONFIG_PATH" but don't actually say what commands you're running or what precisely you're typing. Without those details, it's not really possible to help you.

To get help, please at least provide these things:

* A screenshot of the problem occurring.
* The _precise_ sequence of steps you're taking that results in an error. Give as much detail as possible.

**Comment by yylays** [comment]
thanks for replying.
in windows 10, set system environment varible, my ripgreprc file is adding one file type. sdl
 here is the screen  shot:
![image](https://user-images.githubusercontent.com/11807565/89477510-08d39c80-d7c0-11ea-8780-06818b2bf222.png)

**Comment by BurntSushi** [comment, marked as answer]
I don't know what the OS error message means, but did you see the second line? It's telling you that you're using an unrecognized file type. So... Use a different file type or don't use one at all just to see if things are otherwise working normally.

**Reply by okdana** [reply]
Seems like the problem is that they're setting `RIPGREP_CONFIG_PATH` to a directory rather than the actual file, right?

**Reply by yylays** [reply]
oh yes it is reason. Thanks
